src/components/TimeCardPage/CellTypes/CommentModals/ShowCommentModal.tsx

```diff
@@ -0,0 +1,245 @@
+        getSubmitButtonProps,
+        getCancelButtonProps,
+        getEditButtonProps,
+      } = useEditableControls();
+  
+      // TODO: change this later to reflect figma
+      return isEditing ? (
+        <ButtonGroup justifyContent="center" size="sm">
+          <Button leftIcon={<CheckIcon />} {...getSubmitButtonProps()} />
+          <Button leftIcon={<CloseIcon />} {...getCancelButtonProps()} />
+        </ButtonGroup>
+      ) : (
+        <Flex justifyContent="right">
+          <Button size="sm" leftIcon={<EditIcon />} {...getEditButtonProps()} />
+        </Flex>
+      );
+    };
+  
+    const doCommentsExist = comments.length > 0
+  
+    // no comments so gray it out
+    if (doCommentsExist === false) {
+      color = Color.Gray
+    }
+  
+    const DisplayCommentsModal = () => {
+  
+      return (
+        <Modal isOpen={isOpenDisplay} onClose={onCloseDisplay}>
+          <ModalOverlay />
+          <ModalContent>
+            <ModalHeader>
+              <HStack>
+                <Text>View {CommentType.Comment}</Text>
+                <Button onClick={onOpenAdd}>
+                  New
+                </Button>
+              </HStack>
+            </ModalHeader>
+            <ModalCloseButton />
+            <ModalBody>
+              {comments.map(
+                (comment) => (
+                  <HStack>
+                    {/* add UserDisplay card once pr merged in*/}
+                    <Editable
+                      isDisabled={!isEditable}
+                      defaultValue={comment.Content}
+                      onSubmit={(value) => saveEditedComment(setComments, comments, CommentType.Comment, comment, createNewComment(user, CommentType.Comment, value))}
+                    >
+                      <EditablePreview />
+  
+                      {isEditable && (
+                        <>
+                          <Input as={EditableInput} />
+                          <HStack>
+                            <EditableControls />
+                            <IconButton aria-label="Delete" icon={<DeleteIcon />} onClick={() => deleteComment(onCloseDisplay, setComments, comments, CommentType.Comment, comment)} />
+                          </HStack>
+                        </>
+                      )}
+                    </Editable>
+  
+                  </HStack>
+                ))}
+            </ModalBody>
+  
+            <ModalFooter>
+            </ModalFooter>
+          </ModalContent>
+        </Modal>
+      )
+    }
+  
+    const AddCommentModal = () => {
+      const [remark, setRemark] = useState();
+      const user = useContext(UserContext);
+  
+      const handleRemarkChange = (e) => {
+        setRemark(e.target.value);
+      };
+  
+      const handleSubmit = () => {
+        // TODO: reuse comment validation
+        setComments([...comments, createNewComment(user, CommentType.Comment, remark)]);
+        alert(`Your ${CommentType.Comment} has been submitted!`);
```

Comment:
I think you can remove this alert, might look better without it

Reply:
just there as a placeholder, should be replaced by a toast later on. I'll add a comment to have a toast there instead